src/Contact/FriendSuggest/Repository/FriendSuggest.php

```diff
@@ -9,7 +9,7 @@
 
 use Friendica\BaseRepository;
 use Friendica\Contact\FriendSuggest\Collection;
-use Friendica\Contact\FriendSuggest\Entity;
+use Friendica\Contact\FriendSuggest\Entity\FriendSuggest as EntityFriendSuggest;
```

Comment:
Please rename to `FriendSuggestEntity`
```suggestion
use Friendica\Contact\FriendSuggest\Entity\FriendSuggest as FriendSuggestEntity;
```

Reply:
:+1: Done.